docs/source/config_generated.rst

```diff
@@ -784,9 +784,9 @@ List of input constraints
     If we are trying to constrain only one control (i.e the z control) value:
     | input_constraints:
     | - weights:
-    |   point_3D_0.x-0: 0
-    |   point_3D_0.y-1: 0
-    |   point_3D_0.z-2: 1
+    |   point_3D.x-0: 0
```

Comment:
Could you split this into a separate commit? Fine to include it, but just label it: `example simplification` or something along those lines.